importer: WIP on removing PGDUMP and MYSQLDUMP support

yuzefovich opened the pull request:
Epic: None
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/147191)
<!-- Reviewable:end -->

yuzefovich committed:
importer: inline prepareExistingTablesForIngestion

We now can only import into a single table, so this commit inlines
`prepareExistingTablesForIngestion` under this assumption. An assertion
is added as a sanity check.

Release note: None

yuzefovich committed:
WIP on single table assumption

TODO: add a separate commit with renames of "tables" to "table". Ditto
for comments.
